packages/palette/src/elements/Text/Text.tsx

```diff
@@ -29,8 +30,18 @@ const textColor = style({
   key: "colors",
 })
 
+type TextTransform =
+  | "none"
+  | "capitalize"
+  | "uppercase"
+  | "lowercase"
+  | "initial"
+  | "inherit"
```

Comment:
Could this be pulled from general dom types? `CSS.Something`

Reply:
I can if I pull in https://github.com/frenic/csstype — it's a dependency of styled-system anyway 🤷

Reply:
Oh i see, i was thinking that was a [part of this](https://github.com/artsy/force/blob/master/tsconfig.json#L23)